src/binding/python/ChunkInfo.cpp

```diff
@@ -19,6 +19,7 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 #include "openPMD/ChunkInfo.hpp"
+#include "openPMD/binding/python/Mpi.hpp"
```

Comment:
Does this need an `openPMD_HAVE_MPI` guard?
Please include after `Common.hpp`

Reply:
`.../python/Mpi.hpp` has that guard itself, so it's safe to include unguarded.